docs/current/cli/979595-reference.md

```diff
@@ -92,6 +96,24 @@ dagger run -- sh -c 'curl \
   http://127.0.0.1:$DAGGER_SESSION_PORT/query'
 ```
 
+Output Dagger logs to file:
```

Comment:
I think it's worth describing the simple redirect case too:

    Direct command output to file, displaying progress to terminal:

    ```shell
    dagger run go run ci.go > go-run.out
    ```

Reply:
Added